concurrency-tests/build.gradle

```diff
@@ -5,7 +5,6 @@ plugins {
 dependencies {
     implementation project(":micrometer-core")
 //    implementation("io.micrometer:micrometer-core:1.12.4")
-    implementation project(":micrometer-test")
```

Comment:
This dependency was causing a weird ClassNotFoundException when running jcstress from IntelliJ, but it is only here for the `@Issue` annotation. We do not do any processing of the annotation, so I'm not sure there's any more value of it being an actual annotation vs a comment - I've commented it out where it was used in this module.